typescript/simple-mercari-web/src/components/ItemList/ItemList.tsx

```diff
@@ -17,27 +21,26 @@ interface Prop {
 
 export const ItemList: React.FC<Prop> = (props) => {
   const { reload = true, onLoadCompleted } = props;
-  const [items, setItems] = useState<Item[]>([])
+  const [items, setItems] = useState<Item[]>([]);
   const fetchItems = () => {
```

Comment:
関数内で`await`を使うために`async`をつけておく必要があります！(async, awaitについては後述)
```suggestion
  const fetchItems = async() => {
```